contracts/governance/scripts/AddOperator.sol

```diff
@@ -0,0 +1,34 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.11;
+
+import "../../interfaces/INestedFactory.sol";
+import "../../interfaces/IOperatorResolver.sol";
+import "../../abstracts/MixinOperatorResolver.sol";
+
+contract AddOperator {
+    /// @notice Call NestedFactory and OperatorResolver to add an operator.
+    /// @param nestedFactory The NestedFactory address
+    /// @param operator The operator to add
+    /// @param name The operator bytes32 name
+    function addOperator(
+        address nestedFactory,
+        IOperatorResolver.Operator memory operator,
+        bytes32 name
+    ) external {
+        IOperatorResolver resolver = IOperatorResolver(MixinOperatorResolver(nestedFactory).resolver());
```

Comment:
In case the address of the NestedFactory is not correct, wouldn't it be cheaper in gas to make a **_require_** to check it first?